package.json

```diff
@@ -62,7 +62,6 @@
     "site:build:install": "cd site && npm install --no-audit",
     "site:build:assets": "cd site && npm run build",
     "postinstall-pack": "node ./scripts/postinstall.js",
-    "postinstall": "npm run build && node ./scripts/postinstall.js",
```

Comment:
See comment above – we were also triggering this when running `npm i` in this repo when _developing_ in this repo, but that seems unnecessary so I didn't bother preserving it.

Reply:
... In fact this may be why we had the preinstall build step in the first place... so that this script can run (it uses some npm deps).